TrackCat/settings.py

```diff
@@ -131,7 +132,37 @@
 # Absolute filesystem path to the directory that will hold user-uploaded files.
 MEDIA_ROOT = normpath(join(BASE_DIR, 'media'))
 # URL that handles the media served from MEDIA_ROOT.
-MEDIA_URL = '/media/'
+#MEDIA_URL = '/media/' -> this was used when running app locally
 MEDIA_UPLOAD_FOLDER = 'profile_picture'
 ########## END MEDIA CONFIGURATION
 
+import dj_database_url
+DATABASES['default'] =  dj_database_url.config()
+
+SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
+
+ALLOWED_HOSTS = ['*']
+
+STATIC_ROOT = 'staticfiles'
+
+DEBUG = False
+
+SOCIAL_AUTH_GITHUB_KEY = '77dd707652a6541bf8a4'
```

Comment:
oh, this is nice to know :grin: 
@anuschka I recommend removing this (and other secrets), and then recreating all of them, because these can no longer be considered safe...